pandas/io/parquet.py

```diff
@@ -92,8 +97,7 @@ def write(
         **kwargs,
     ):
         self.validate_dataframe(df)
-        path, _, _, _ = get_filepath_or_buffer(path, mode="wb")
-
+        file_obj, _, _, _ = get_filepath_or_buffer(path, mode="wb")
```

Comment:
@jorisvandenbossche think we can clean up the `write` method here to get rid of `get_filepath_or_buffer` similar to what i've done below for `read`. Will address in different PR.